docs/remix_as_code_viewer.md

```diff
@@ -0,0 +1,38 @@
+Remix as code viewer
+=============================
+
+Through Etherscan
+------------------
+
+If there is a contract verified on Etherscan and someone wants to have a look on it in an IDE, mostly for a mutiple part contract verification, Remix provides a quick way to load the contract.
+
+Usually, contract is shown on Etherscan on a URL like this:
+
+`https://etherscan.io/address/0xdac17f958d2ee523a2206206994597c13d831ec7`
+
+In URL, just update `etherscan.io` with `remix.ethereum.org`
+
+`https://remix.ethereum.org/address/0xdac17f958d2ee523a2206206994597c13d831ec7`
+
+and reload. It will fetch the contracts verified on Ethereum mainnnet and on other testnets (Ropsten, Rinkeby, Kovan & Goerli).
+
+![](images/a-code-viewer-etherscan.png)
+
+This works for Etherscan testnet URLs like `https://ropsten.etherscan.io` etc.
```

Comment:
this line is basically a repeat of line 17.

Reply:
17 one says that contracts will be loaded from different testnets and this says that you can also update testnet urls too

Reply:
I update to make it more clear